interfaces/builtin/anbox_support.go

```diff
@@ -0,0 +1,512 @@
+    deny /proc/sys/kernel/domainn[^a]*{,/**} wklx,
+    deny /proc/sys/kernel/domainna[^m]*{,/**} wklx,
+    deny /proc/sys/kernel/domainnam[^e]*{,/**} wklx,
+    deny /proc/sys/kernel/domainname?*{,/**} wklx,
+    deny /proc/sys/kernel/h[^o]*{,/**} wklx,
+    deny /proc/sys/kernel/ho[^s]*{,/**} wklx,
+    deny /proc/sys/kernel/hos[^t]*{,/**} wklx,
+    deny /proc/sys/kernel/host[^n]*{,/**} wklx,
+    deny /proc/sys/kernel/hostn[^a]*{,/**} wklx,
+    deny /proc/sys/kernel/hostna[^m]*{,/**} wklx,
+    deny /proc/sys/kernel/hostnam[^e]*{,/**} wklx,
+    deny /proc/sys/kernel/hostname?*{,/**} wklx,
+    deny /proc/sys/kernel/m[^s]*{,/**} wklx,
+    deny /proc/sys/kernel/ms[^g]*{,/**} wklx,
+    deny /proc/sys/kernel/msg*/** wklx,
+    deny /proc/sys/kernel/s[^he]*{,/**} wklx,
+    deny /proc/sys/kernel/se[^m]*{,/**} wklx,
+    deny /proc/sys/kernel/sem*/** wklx,
+    deny /proc/sys/kernel/sh[^m]*{,/**} wklx,
+    deny /proc/sys/kernel/shm*/** wklx,
+    deny /proc/sys/kernel?*{,/**} wklx,
+    deny /proc/sys/n[^e]*{,/**} wklx,
+    deny /proc/sys/ne[^t]*{,/**} wklx,
+    deny /proc/sys/net?*{,/**} wklx,
+    deny /sys/[^fdck]*{,/**} wklx,
+    deny /sys/c[^l]*{,/**} wklx,
+    deny /sys/cl[^a]*{,/**} wklx,
+    deny /sys/cla[^s]*{,/**} wklx,
+    deny /sys/clas[^s]*{,/**} wklx,
+    deny /sys/class/[^n]*{,/**} wklx,
+    deny /sys/class/n[^e]*{,/**} wklx,
+    deny /sys/class/ne[^t]*{,/**} wklx,
+    deny /sys/class/net?*{,/**} wklx,
+    deny /sys/class?*{,/**} wklx,
+    deny /sys/d[^e]*{,/**} wklx,
+    deny /sys/de[^v]*{,/**} wklx,
+    deny /sys/dev[^i]*{,/**} wklx,
+    deny /sys/devi[^c]*{,/**} wklx,
+    deny /sys/devic[^e]*{,/**} wklx,
+    deny /sys/device[^s]*{,/**} wklx,
+    deny /sys/devices/[^v]*{,/**} wklx,
+    deny /sys/devices/v[^i]*{,/**} wklx,
+    deny /sys/devices/vi[^r]*{,/**} wklx,
+    deny /sys/devices/vir[^t]*{,/**} wklx,
+    deny /sys/devices/virt[^u]*{,/**} wklx,
+    deny /sys/devices/virtu[^a]*{,/**} wklx,
+    deny /sys/devices/virtua[^l]*{,/**} wklx,
+    deny /sys/devices/virtual/[^n]*{,/**} wklx,
+    deny /sys/devices/virtual/n[^e]*{,/**} wklx,
+    deny /sys/devices/virtual/ne[^t]*{,/**} wklx,
+    deny /sys/devices/virtual/net?*{,/**} wklx,
+    deny /sys/devices/virtual?*{,/**} wklx,
+    deny /sys/devices?*{,/**} wklx,
+    deny /sys/f[^s]*{,/**} wklx,
+    deny /sys/fs/[^c]*{,/**} wklx,
+    deny /sys/fs/c[^g]*{,/**} wklx,
+    deny /sys/fs/cg[^r]*{,/**} wklx,
+    deny /sys/fs/cgr[^o]*{,/**} wklx,
+    deny /sys/fs/cgro[^u]*{,/**} wklx,
+    deny /sys/fs/cgrou[^p]*{,/**} wklx,
+    deny /sys/fs/cgroup?*{,/**} wklx,
+    deny /sys/fs?*{,/**} wklx,
+
+    ### Feature: unix
+    # Allow receive via unix sockets from anywhere
+    unix (receive),
+
+    # Allow all unix in the container
+    unix peer=(label=@{profile_name}),
+
+    ### Feature: cgroup namespace
+    mount fstype=cgroup -> /sys/fs/cgroup/**,
+
+    ### Configuration: nesting
+    pivot_root,
+    ptrace,
+    signal,
```

Comment:
You aren't doing nesting are you? This ptrace and signal rule masks all the others.